Utilities/DataSampling/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
+# under the terms of the GNU General Public License v3 (GPL Version 3), copied
+# verbatim in the file "COPYING".
+#
+# See http://alice-o2.web.cern.ch/license for full licensing information.
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+# FIXME: the LinkDef should not be in the public area
+
+o2_add_library(DataSampling
+               SOURCES   src/DataSampling.cxx
+                         src/DataSamplingConditionFactory.cxx
+                         src/DataSamplingHeader.cxx
+                         src/DataSamplingConditionCustom.cxx
+                         src/DataSamplingConditionNConsecutive.cxx
+                         src/DataSamplingConditionPayloadSize.cxx
+                         src/DataSamplingConditionRandom.cxx
+                         src/DataSamplingHeader.cxx
+                         src/DataSamplingPolicy.cxx
+                         src/DataSamplingReadoutAdapter.cxx
+                         src/Dispatcher.cxx
+
+  PUBLIC_LINK_LIBRARIES O2::Framework O2::DataSampling)
```

Comment:
I am not sure about declaring DataSampling as dependency of DataSampling.

Reply:
indeed, this is wrong